includes/embeds/class-amp-twitter-embed-handler.php

```diff
@@ -161,9 +181,9 @@ private function create_amp_twitter_and_replace_node( Document $dom, DOMElement
 		}
 
 		$attributes = [
-			'width'        => $this->DEFAULT_WIDTH,
+			'width'        => 'auto',
```

Comment:
Any reason for this change?

```suggestion
			'width'        => $this->DEFAULT_WIDTH,
```

Reply:
Yes, because `auto` is the only valid width for the `fixed-height` layout.

Reply:
Oh wait, you're right. I've revised in https://github.com/ampproject/amp-wp/pull/6510